### PR TITLE
Feature: about me, my skills and my works

### PR DESCRIPTION
**Pull request by Edjhernandez**
# Here! i did sections about me, my skills and my works 🔢 for mobile version 📲 

## Profile photo 📸 
profile

## my description 📝 
my description

## my skills 🧰 
myskills

## my works! 🧑‍💼
myworks

### CODE REVIEW

**Comment by Edjhernandez** [comment]
Hi estimed @Albinpirela, here are the last changes, please in the review take special atention with the html semantic. i know you have a lot of knowledge on that area. 👍

File: index.html [hunk 1]
```diff
@@ -27,13 +27,119 @@
...
+            &nbsp;}<br>
+            &nbsp;writeSymbols();<br>
+            } ...</p>
+        </div>  
+        <div class="photo">
+          <div class="photolayer1"></div>
+          <div class="photolayer2"></div>
+          <div class="photolayer3"></div>
+        </div>
+      </div>
```
Comment (Albinpirela) [inline index.html]: Hi, @Edjhernandez  good job, your work is excellent. I'm seeing some empty labels, can you tell me what their purpose is?

File: index.html [hunk 2]
```diff
@@ -27,13 +27,119 @@
...
+        </div>
+        <p>A few years ago, I became interested in developing websites and interfaces.
+        Since then, I have been steadily improving my skills. Now I am developing websites
+        and web applications to order or for myself.</p>
+      </div>
+      <div class="myskills">
+        <h2>My Skills</h2>
+        <div class="skill">
+          <img src="./images/icon html.png" alt="HTML5 icon">
+          <img class="top" src="./images/icon html -- hover.png" alt="HTML5 icon hover">
```
Comment (Albinpirela) [inline index.html]: I was also seeing that you have a couple of imgs that contain the same image, I suggest you use an svg image so you can change the color of this icon

File: index.html [hunk 3]
```diff
@@ -27,13 +27,119 @@
...
+          <p>HTML5</p>
+        </div>
+        <div class="skill">
+          <img src="./images/Exclude (1).png" alt="CSS icon">
+          <img class="top" src="./images/css hover.png" alt="CSS icon hover">
```
Comment (Albinpirela) [inline index.html]: the same way bellow

File: index.html [hunk 4]
```diff
@@ -27,13 +27,119 @@
...
+          <p>CSS</p>
+        </div>
+        <div class="skill">
+          <img src="./images/Exclude.png" alt="Js icon">
+          <img class="top" src="./images/icon js -- hover.png" alt="Js icon hover">
```
Comment (Albinpirela) [inline index.html]: the same way bellow

File: index.html [hunk 5]
```diff
@@ -27,13 +27,119 @@
...
+          <p>JavaScript</p>
+        </div>
+        <div class="skill">
+          <img src="./images/icon scss.png" alt="SCSS icon">
+          <img class="top" src="./images/icon scss -- hover.png" alt="SCSS icon hover">
```
Comment (Albinpirela) [inline index.html]: the same way bellow

File: index.html [hunk 6]
```diff
@@ -27,13 +27,119 @@
...
+          <p>SCSS</p>
+        </div>
+        <div class="skill">
+          <img src="./images/Exclude (2).png" alt="Git icon">
+          <img class="top" src="./images/icon git -- hover.png" alt="Git icon hover">
```
Comment (Albinpirela) [inline index.html]: the same way bellow

**Review by Albinpirela — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Edjhernandez) [reply index.html]: Hi estimed @Albinpirela, thanks for your suggestions.
These empty labels were created to make the polygons behind the profile photo.
In the styles.css we can find the class and see how css creates the polygons.

**Review by Albinpirela — COMMENTED** [review]
Good your work is very good it is approvedery good your work is very good it is approved

File: styles.css [hunk 7]
```diff
@@ -43,10 +45,10 @@ div.logo :nth-child(2) {
 }
 
 button.menubutton {
-  margin-right: 8px;
+  margin-right: 0.5rem;
   align-items: center;
-  width: 48px;
-  height: 48px;
+  width: 3rem;
+  height: 3rem;
```
Comment (Albinpirela) [inline styles.css]: I suggest that you change the unit of measurement of the width and use % for better results in the responsive part 😃

File: styles.css [hunk 8]
```diff
@@ -67,28 +69,205 @@ span.material-symbols-outlined {
...
+  margin: 0;
 }
 
 div.headline :nth-child(2) {
   font-weight: 700;
   font-size: 2rem;
   line-height: 34px;
   color: var(--text);
-  margin: 16px 0 16px 68px;
+  margin: 1rem 0;
```
Comment (Albinpirela) [inline styles.css]: I suggest you use (em) for the margin

**Review by Albinpirela — CHANGES_REQUESTED** [review]
Hello @Edjhernandez  you have done an excellent job, I suggest taking care of the responsive theme 💹 
Captura de pantalla (52)

File: index.html (hunk 2 above)
Comment (Edjhernandez) [reply index.html]: Hi @Albinpirela thanks for the suggestion, i'll change that. 👍

File: styles.css (hunk 7 above)
Comment (Edjhernandez) [reply styles.css]: Hi @Albinpirela, thanks for the suggestion, i'll make this change. 🏃

File: styles.css (hunk 8 above)
Comment (Edjhernandez) [reply styles.css]: Thanks @Albinpirela i'll do it.

**Comment by Edjhernandez** [comment]
Hello estimed @Albinpirela, i think all your suggestions were made:

1. All units of margin and padding are in **em**.
2. Properties of width and height are in **%**.
3. I've had special attention on responsive design.
4. All units of font-size are in **rem**.
5. I changed the skills images for **svg icons**.

Screenshot 2023-03-20 055438

**Review by Albinpirela — APPROVED** [review]
Approved @Edjhernandez good job you can merge. 👍 👷

**Pull request merged**